packages/gatsby-theme-carbon/src/components/Header/Header.module.scss

```diff
@@ -121,3 +192,37 @@ header.header {
     border-left: 1px solid $carbon--white-0;
   }
 }
+
+.header :global(a.bx--header__menu-item:hover) {
+  color: $carbon--white-0;
+}
+
+// Style for the subtle border under each item in the headernav sub menu.
```

Comment:
In general we want comments to explain **why** something was done a certain way. If you find yourself explaining **what** is happening, there's a good chance the code could be clearer.